src/views/InvolvementProfile/components/Membership/components/AdminCard/components/RequestsReceived/RequestsReceived.module.scss

```diff
@@ -1,5 +1,5 @@
 .redButton {
-  color: var(--mui-palette-error-main);
+  color: var(--mui-palette-error-dark);
```

Comment:
If you want this to be red, can you use ```color="red"``` on the button instead, especially if you make OldSchemeRed error-main again